src/dotnet-roslyn-tools/Commands/PRTaggerCommand.cs

```diff
@@ -2,29 +2,27 @@
 // The.NET Foundation licenses this file to you under the MIT license.
 // See the License.txt file in the project root for more information.
 
-namespace Microsoft.RoslynTools.Commands;
-
-using System.CommandLine.Invocation;
 using System.CommandLine;
+using System.CommandLine.Invocation;
+using System.Net.Http.Headers;
 using Microsoft.Extensions.Logging;
-using Microsoft.RoslynTools.Products;
+using Microsoft.RoslynTools.Utilities;
+
+namespace Microsoft.RoslynTools.Commands;
 
 internal static class PRTaggerCommand
 {
     private static readonly PRTaggerCommandDefaultHandler s_prTaggerCommandHandler = new();
-
-    private static readonly Option<string> VSBuild = new(new[] { "--build", "-b" }, "VS build number") { IsRequired = true };
-    private static readonly Option<string> CommitId = new(new[] { "--commit", "-c" }, "VS build commit") { IsRequired = true };
+    public static readonly Option<int> maxVsBuildCheckNumber = new(new[] { "--vsBuildCheckNumber" }, () => 20, "Maximum number of VS build to check. Tagger would compare each VS build and its parent commit to find the inserted payload.");
```

Comment:
What is the most builds that they've had in a day? Will 40 be enough?

Reply:
It generated about 30 per day. but I will increase this number to 50 to make sure things are correct